sdk/storage/azure-storage-blob/tests/perfstress_tests/list_blobs.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import itertools
+import asyncio
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from ._test_base import _ContainerTest
+
+
+class ListBlobsTest(_ContainerTest):
+
+    async def global_setup(self):
+        await super().global_setup()
+        pending = (asyncio.ensure_future(self.async_container_client.upload_blob("listtest" + str(i), data=b"")) for i in range(self.args.num_blobs))
```

Comment:
I'm not sure how this works, but if it is creating the blobs in parallel and it works fine, it seems good to me :).

Reply:
Yup - this just keeps the maximum concurrent operations at 16, and keeps adding new uploads to the pool when one finishes.